### ISSUE TITLE
SVG issues after upgrade

### ISSUE THREAD

**Issue by Lartza** [nextcloud/server#2564]
### Steps to reproduce
1. Production 10.0.1 server
2. Upgrade to 10.0.2 RC2 using the web updater
3. Upgrade to 11.0 RC1 using the web updater

### Expected behaviour
Folders have a folder icon shown before them, no errors about SVG images.

### Actual behaviour
Folders miss the folder.svg icon and instead a blank space is shown. Log has an error after the update and shows errors weirdly enough for audio.svg. audio.svg error showed constantly browsing the Files app and was logged last ~20 minutes before changing colors in the Theming app so assuming it was fixed by that action.

data/appdata_xxxx/theming/1/icon-core-filetypes_folder.svg is an empty file. Still gets served correctly as image/svg+xml so no errors in nginx or browser. Browsing /apps/theming/img/core/filetypes/folder.svg?v=X makes the browser to try and interpret the 0 bytes as XML.

folder.svg error only appeared ONCE just before regenerating because I deleted data/appdata_xxxx/theming/1/icon-core-filetypes_folder.svg to try and regenerate it before changing color in the Theming app.

Going to theming and changing the color to something else and back regenerated the folder.svg succesfully. audio.svg error stopped happening little before this action as per above.

### Server configuration
**Operating system**: Arch Linux

**Web server:** nginx 1.11.6

**Database:** PostgreSQL 9.6.1

**PHP version:** 7.0.13

**Nextcloud version:** 11.0 RC1

**Updated from an older Nextcloud/ownCloud or fresh install:** Upgrade 10.0.2RC2 > 11 RC1 (also 10.0.1 > 10.0.2 RC2 before that but error appeared in 11)

**Where did you install Nextcloud from:** zip or bz2

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
  - activity: 2.4.1
  - comments: 1.1.0
  - dav: 1.1.1
  - federatedfilesharing: 1.1.1
  - federation: 1.1.1
  - files: 1.6.1
  - files_pdfviewer: 1.0.1
  - files_sharing: 1.1.1
  - files_texteditor: 2.2
  - files_trashbin: 1.1.0
  - files_versions: 1.4.0
  - files_videoplayer: 1.0.0
  - firstrunwizard: 2.0
  - gallery: 16.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - nextcloud_announcements: 1.0
  - notifications: 1.0.1
  - password_policy: 1.1.0
  - provisioning_api: 1.1.0
  - serverinfo: 1.1.1
  - sharebymail: 1.0.1
  - survey_client: 0.1.5
  - systemtags: 1.1.3
  - theming: 1.1.1
  - twofactor_backupcodes: 1.0.0
  - updatenotification: 1.1.1
  - workflowengine: 1.1.1
Disabled:
  - admin_audit
  - encryption
  - external
  - files_accesscontrol
  - files_automatedtagging
  - files_external
  - files_retention
  - templateeditor
  - user_external
  - user_ldap
  - user_saml
```
</details>

**The content of config/config.php:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "ocsvvyxnbhp9",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.mydomain.com"
        ],
        "datadirectory": "\/usr\/share\/nginx\/cloud.mydomain.com\/data",
        "overwrite.cli.url": "https:\/\/cloud.mydomain.com",
        "dbtype": "pgsql",
        "version": "11.0.0.6",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_from_address": "no-reply",
        "mail_domain": "mydomain.com",
        "mail_smtpsecure": "tls",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "smtp.gmail.com",
        "mail_smtpport": "587",
        "maintenance": false
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "localhost",
            "port": 6379
        }
    }
}
```
</details>

**Are you using external storage, if yes which one:** No

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:** Firefox 50.0.2

**Operating system:** Windows 10

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
{"reqId":"497BHFXNco660Zuqtzm+","remoteAddr":"x.x.x.x","app":"updater","message":"\\OC\\Updater::updateEnd: Update successful","level":1,"time":"2016-12-07T16:39:07+00:00","method":"GET","url":"\/core\/ajax\/update.php?requesttoken=xltdw7WP2k2Qc5gmr8yDOn2i8Uc8yRDrd46rk7XcKck%3D%3AjhwYsfb5lX%2B%2FNKpNwofidCnPhwUNo0CxL7iT%2BOOZQ%2F0%3D","user":"--","version":"11.0.0.6"}
{"reqId":"497BHFXNco660Zuqtzm+","remoteAddr":"x.x.x.x","app":"updater","message":"\\OC\\Updater::maintenanceDisabled: Turned off maintenance mode","level":1,"time":"2016-12-07T16:39:07+00:00","method":"GET","url":"\/core\/ajax\/update.php?requesttoken=xltdw7WP2k2Qc5gmr8yDOn2i8Uc8yRDrd46rk7XcKck%3D%3AjhwYsfb5lX%2B%2FNKpNwofidCnPhwUNo0CxL7iT%2BOOZQ%2F0%3D","user":"--","version":"11.0.0.6"}
{"reqId":"497BHFXNco660Zuqtzm+","remoteAddr":"x.x.x.x","app":"updater","message":"\\OC\\Updater::resetLogLevel: Reset log level to Warning(2)","level":1,"time":"2016-12-07T16:39:07+00:00","method":"GET","url":"\/core\/ajax\/update.php?requesttoken=xltdw7WP2k2Qc5gmr8yDOn2i8Uc8yRDrd46rk7XcKck%3D%3AjhwYsfb5lX%2B%2FNKpNwofidCnPhwUNo0CxL7iT%2BOOZQ%2F0%3D","user":"--","version":"11.0.0.6"}
{"reqId":"tUo388lxDLXtkEm9i0\/k","remoteAddr":"x.x.x.x","app":"index","message":"Exception: {\"Exception\":\"Doctrine\\\\DBAL\\\\Exception\\\\UniqueConstraintViolationException\",\"Message\":\"An exception occurred while executing 'INSERT INTO \\\"oc_filecache\\\" (\\\"mimepart\\\",\\\"mimetype\\\",\\\"mtime\\\",\\\"size\\\",\\\"etag\\\",\\\"storage_mtime\\\",\\\"permissions\\\",\\\"parent\\\",\\\"checksum\\\",\\\"path_hash\\\",\\\"path\\\",\\\"name\\\",\\\"storage\\\") SELECT ?,?,?,?,?,?,?,?,?,?,?,?,? FROM \\\"oc_filecache\\\" WHERE \\\"storage\\\" = ? AND \\\"path_hash\\\" = ? HAVING COUNT(*) = 0' with params [1, 2, 1481128768, -1, \\\"58483b40ef144\\\", 1481128768, 31, 15001, \\\"\\\", \\\"e6acd61ab16c4a02eeb2b7cdda6d90fb\\\", \\\"appdata_ocsvvyxnbhp9\\\\\\\/preview\\\", \\\"preview\\\", 2, 2, \\\"e6acd61ab16c4a02eeb2b7cdda6d90fb\\\"]:\\n\\nSQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint \\\"fs_storage_path_hash\\\"\\nDETAIL:  Key (storage, path_hash)=(2, e6acd61ab16c4a02eeb2b7cdda6d90fb) already exists.\",\"Code\":0,\"Trace\":\"#0 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/3rdparty\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/DBALException.php(128): Doctrine\\\\DBAL\\\\Driver\\\\AbstractPostgreSQLDriver->convertException('An exception oc...', Object(Doctrine\\\\DBAL\\\\Driver\\\\PDOException))\\n#1 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/3rdparty\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/Connection.php(996): Doctrine\\\\DBAL\\\\DBALException::driverExceptionDuringQuery(Object(Doctrine\\\\DBAL\\\\Driver\\\\PDOPgSql\\\\Driver), Object(Doctrine\\\\DBAL\\\\Driver\\\\PDOException), 'INSERT INTO \\\"oc...', Array)\\n#2 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/DB\\\/Connection.php(215): Doctrine\\\\DBAL\\\\Connection->executeUpdate('INSERT INTO \\\"oc...', Array, Array)\\n#3 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/DB\\\/Adapter.php(114): OC\\\\DB\\\\Connection->executeUpdate('INSERT INTO \\\"oc...', Array)\\n#4 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/DB\\\/Connection.php(253): OC\\\\DB\\\\Adapter->insertIfNotExist('*PREFIX*filecac...', Array, Array)\\n#5 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Cache\\\/Cache.php(276): OC\\\\DB\\\\Connection->insertIfNotExist('*PREFIX*filecac...', Array, Array)\\n#6 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Cache\\\/Cache.php(233): OC\\\\Files\\\\Cache\\\\Cache->insert('appdata_ocsvvyx...', Array)\\n#7 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Cache\\\/Scanner.php(277): OC\\\\Files\\\\Cache\\\\Cache->put('appdata_ocsvvyx...', Array)\\n#8 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Cache\\\/Scanner.php(217): OC\\\\Files\\\\Cache\\\\Scanner->addToCache('appdata_ocsvvyx...', Array, -1)\\n#9 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Cache\\\/Scanner.php(320): OC\\\\Files\\\\Cache\\\\Scanner->scanFile('appdata_ocsvvyx...', 3, 15001, false, true)\\n#10 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/View.php(1301): OC\\\\Files\\\\Cache\\\\Scanner->scan('appdata_ocsvvyx...', false)\\n#11 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/View.php(1342): OC\\\\Files\\\\View->getCacheEntry(Object(OCA\\\\Files_Trashbin\\\\Storage), 'appdata_ocsvvyx...', '\\\/appdata_ocsvvy...')\\n#12 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Node\\\/Root.php(196): OC\\\\Files\\\\View->getFileInfo('\\\/appdata_ocsvvy...')\\n#13 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Node\\\/Folder.php(122): OC\\\\Files\\\\Node\\\\Root->get('\\\/appdata_ocsvvy...')\\n#14 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/AppData\\\/AppData.php(89): OC\\\\Files\\\\Node\\\\Folder->get('preview')\\n#15 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/AppData\\\/AppData.php(105): OC\\\\Files\\\\AppData\\\\AppData->getAppDataFolder()\\n#16 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Preview\\\/Watcher.php(61): OC\\\\Files\\\\AppData\\\\AppData->getFolder(15007)\\n#17 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Preview\\\/WatcherConnector.php(60): OC\\\\Preview\\\\Watcher->postWrite(Object(OC\\\\Files\\\\Node\\\\File))\\n#18 [internal function]: OC\\\\Preview\\\\WatcherConnector->OC\\\\Preview\\\\{closure}(Object(OC\\\\Files\\\\Node\\\\File))\\n#19 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Hooks\\\/EmitterTrait.php(99): call_user_func_array(Object(Closure), Array)\\n#20 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Hooks\\\/PublicEmitter.php(33): OC\\\\Hooks\\\\BasicEmitter->emit('\\\\\\\\OC\\\\\\\\Files', 'postWrite', Array)\\n#21 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Node\\\/Root.php(134): OC\\\\Hooks\\\\PublicEmitter->emit('\\\\\\\\OC\\\\\\\\Files', 'postWrite', Array)\\n#22 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/Node\\\/Folder.php(172): OC\\\\Files\\\\Node\\\\Root->emit('\\\\\\\\OC\\\\\\\\Files', 'postWrite', Array)\\n#23 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Files\\\/SimpleFS\\\/SimpleFolder.php(83): OC\\\\Files\\\\Node\\\\Folder->newFile('icon-core-filet...')\\n#24 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/apps\\\/theming\\\/lib\\\/ImageManager.php(94): OC\\\\Files\\\\SimpleFS\\\\SimpleFolder->newFile('icon-core-filet...')\\n#25 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/apps\\\/theming\\\/lib\\\/Controller\\\/IconController.php(97): OCA\\\\Theming\\\\ImageManager->setCachedImage('icon-core-filet...', '<svg xmlns=\\\"htt...')\\n#26 [internal function]: OCA\\\\Theming\\\\Controller\\\\IconController->getThemedIcon('core', 'filetypes\\\/folde...')\\n#27 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(160): call_user_func_array(Array, Array)\\n#28 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(90): OC\\\\AppFramework\\\\Http\\\\Dispatcher->executeController(Object(OCA\\\\Theming\\\\Controller\\\\IconController), 'getThemedIcon')\\n#29 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/AppFramework\\\/App.php(114): OC\\\\AppFramework\\\\Http\\\\Dispatcher->dispatch(Object(OCA\\\\Theming\\\\Controller\\\\IconController), 'getThemedIcon')\\n#30 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/AppFramework\\\/Routing\\\/RouteActionHandler.php(47): OC\\\\AppFramework\\\\App::main('OCA\\\\\\\\Theming\\\\\\\\Con...', 'getThemedIcon', Object(OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer), Array)\\n#31 [internal function]: OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler->__invoke(Array)\\n#32 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/private\\\/Route\\\/Router.php(299): call_user_func(Object(OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler), Array)\\n#33 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/lib\\\/base.php(1010): OC\\\\Route\\\\Router->match('\\\/apps\\\/theming\\\/i...')\\n#34 \\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/index.php(40): OC::handleRequest()\\n#35 {main}\",\"File\":\"\\\/usr\\\/share\\\/nginx\\\/cloud.mydomain.com\\\/3rdparty\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/Driver\\\/AbstractPostgreSQLDriver.php\",\"Line\":63}","level":3,"time":"2016-12-07T16:39:29+00:00","method":"GET","url":"\/apps\/theming\/img\/core\/filetypes\/folder.svg?v=undefined","user":"Lartza","version":"11.0.0.6"}
{"reqId":"Bo9thqTxH1mF8bsjF+Jy","remoteAddr":"x.x.x.x","app":"PHP","message":"filesize(): stat failed for \/usr\/share\/nginx\/cloud.mydomain.com\/data\/Lartza\/files\/usr\/share\/nginx\/cloud.mydomain.com\/core\/img\/filetypes\/audio.svg at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#211","level":3,"time":"2016-12-07T16:43:21+00:00","method":"GET","url":"\/core\/preview.png?file=%2FLHS_RLD10+.MP3&c=2015846ec1891d72a19478ba5c31052c&x=32&y=32&forceIcon=0","user":"Lartza","version":"11.0.0.6"}
{"reqId":"Bo9thqTxH1mF8bsjF+Jy","remoteAddr":"x.x.x.x","app":"PHP","message":"fopen(\/usr\/share\/nginx\/cloud.mydomain.com\/data\/Lartza\/files\/usr\/share\/nginx\/cloud.mydomain.com\/core\/img\/filetypes\/audio.svg): failed to open stream: No such file or directory at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#216","level":3,"time":"2016-12-07T16:43:21+00:00","method":"GET","url":"\/core\/preview.png?file=%2FLHS_RLD10+.MP3&c=2015846ec1891d72a19478ba5c31052c&x=32&y=32&forceIcon=0","user":"Lartza","version":"11.0.0.6"}
{"reqId":"Bo9thqTxH1mF8bsjF+Jy","remoteAddr":"x.x.x.x","app":"PHP","message":"fread() expects parameter 1 to be resource, boolean given at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#217","level":3,"time":"2016-12-07T16:43:21+00:00","method":"GET","url":"\/core\/preview.png?file=%2FLHS_RLD10+.MP3&c=2015846ec1891d72a19478ba5c31052c&x=32&y=32&forceIcon=0","user":"Lartza","version":"11.0.0.6"}
{"reqId":"Bo9thqTxH1mF8bsjF+Jy","remoteAddr":"x.x.x.x","app":"PHP","message":"fclose() expects parameter 1 to be resource, boolean given at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#218","level":3,"time":"2016-12-07T16:43:21+00:00","method":"GET","url":"\/core\/preview.png?file=%2FLHS_RLD10+.MP3&c=2015846ec1891d72a19478ba5c31052c&x=32&y=32&forceIcon=0","user":"Lartza","version":"11.0.0.6"}
{"reqId":"Bo9thqTxH1mF8bsjF+Jy","remoteAddr":"x.x.x.x","app":"PHP","message":"imagecreatefromstring(): Empty string or invalid image at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/legacy\/image.php#597","level":3,"time":"2016-12-07T16:43:21+00:00","method":"GET","url":"\/core\/preview.png?file=%2FLHS_RLD10+.MP3&c=2015846ec1891d72a19478ba5c31052c&x=32&y=32&forceIcon=0","user":"Lartza","version":"11.0.0.6"}

{"reqId":"aHLJQPVho9VpC5+qzqEG","remoteAddr":"x.x.x.x","app":"PHP","message":"filesize(): stat failed for \/usr\/share\/nginx\/cloud.mydomain.com\/data\/appdata_ocsvvyxnbhp9\/theming\/1\/icon-core-filetypes_folder.svg at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#211","level":3,"time":"2016-12-07T21:33:30+00:00","method":"GET","url":"\/apps\/theming\/img\/core\/filetypes\/folder.svg?v=1","user":"Lartza","version":"11.0.0.6"}
{"reqId":"aHLJQPVho9VpC5+qzqEG","remoteAddr":"x.x.x.x","app":"PHP","message":"fopen(\/usr\/share\/nginx\/cloud.mydomain.com\/data\/appdata_ocsvvyxnbhp9\/theming\/1\/icon-core-filetypes_folder.svg): failed to open stream: No such file or directory at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#216","level":3,"time":"2016-12-07T21:33:30+00:00","method":"GET","url":"\/apps\/theming\/img\/core\/filetypes\/folder.svg?v=1","user":"Lartza","version":"11.0.0.6"}
{"reqId":"aHLJQPVho9VpC5+qzqEG","remoteAddr":"x.x.x.x","app":"PHP","message":"fread() expects parameter 1 to be resource, boolean given at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#217","level":3,"time":"2016-12-07T21:33:30+00:00","method":"GET","url":"\/apps\/theming\/img\/core\/filetypes\/folder.svg?v=1","user":"Lartza","version":"11.0.0.6"}
{"reqId":"aHLJQPVho9VpC5+qzqEG","remoteAddr":"x.x.x.x","app":"PHP","message":"fclose() expects parameter 1 to be resource, boolean given at \/usr\/share\/nginx\/cloud.mydomain.com\/lib\/private\/Files\/Storage\/Local.php#218","level":3,"time":"2016-12-07T21:33:30+00:00","method":"GET","url":"\/apps\/theming\/img\/core\/filetypes\/folder.svg?v=1","user":"Lartza","version":"11.0.0.6"}
```
</details>

**Comment by jancborchardt** [comment]
cc @juliushaertl any idea bere?

**Comment by juliushaertl** [comment]
@Lartza @jancborchardt will have a look later :wink:

**Comment by juliushaertl** [comment]
@Lartza can you check if the icon files in `core/img/filetypes/` are readable by the webserver?

The audio.svg errors seem to be related to the preview rendering.

I currently can't find any issue on the icon generation, also a upgrade from 10.0.1 to 11 RC1  like you've described works fine here with the icons, but we should make sure that issues will not store empty files. I'll prepare a PR for that.

**Comment by Lartza** [comment]
@juliushaertl The icon files are readable and the folders up to there are executable.
Did the PostgreSQL error play any part in folder.svg being empty maybe?

I'm guessing this needs some specific circumstances to be reproduced since it was a production server I upgraded.

Writing above I realized I took a database and file backup before I started upgrading back in 10.0.1 so I could try to reproduce this with some work and I might just do that later, didn't think of that before.
